src/SprykerSdk/Zed/AppSdk/Business/AsyncApi/Builder/AsyncApiBuilder.php

```diff
@@ -390,4 +389,28 @@ protected function orderAsyncApiElements(array $asyncApi): array
 
         return $asyncApi;
     }
+
+    /**
+     * @param string $targetFile
+     * @param array $asyncApi
+     *
+     * @return void
+     */
+    private function updateFile(string $targetFile, array $asyncApi): void
+    {
+        if(file_exists($targetFile)) {
```

Comment:
You already check the existence in https://github.com/spryker-sdk/app-sdk/pull/6/files#diff-9e9f55553f88e7c9e416b816fd28cff68da9179e748e60c5b644820df3314971R45

No need to do it a second time. Filesystem access is slow by definition, avoid as much as you can.